community/consistency-check/src/main/java/org/neo4j/consistency/checking/SchemaRecordCheck.java

```diff
@@ -179,12 +170,26 @@ else if ( phase == Phase.CHECK_OBLIGATIONS )
         }
     }
 
-    private void checkMandatoryProperty( int propertyKey, RecordAccess records,
+    private void checkMandatoryNodePropertyRule( MandatoryNodePropertyConstraintRule rule, DynamicRecord record,
+            RecordAccess records, CheckerEngine<DynamicRecord,ConsistencyReport.SchemaConsistencyReport> engine )
+    {
+        if ( phase == Phase.CHECK_RULES )
```

Comment:
Would agree it's has nothing to do to this PR, but still. Why those `if`s are not represented as phase specific check strategy?